projects/plugins/jetpack/_inc/client/newsletter/index.jsx

```diff
@@ -48,6 +49,7 @@ function Subscriptions( props ) {
 				<SubscriptionsSettings siteRawUrl={ siteRawUrl } blogID={ blogID } />
 			) }
 			{ isNewsletterCategoriesEnabled && <NewsletterCategories /> }
+			<EmailWelcomeMessageSetting { ...props } />
```

Comment:
Let's call it `Messages` or `MessagesSettings` because the settings card may as well include other email messages in the future.

Reply:
Good point. I'll change it.